src/utils/common.utils.ts

```diff
@@ -266,16 +266,16 @@ export async function createClaim(
 }
 
 export function hasScope(user: any, role: string): boolean {
-  console.log('hasScope', user.scopes.auth, role);
   if (!user) return false;
   if (!user.scopes) return false;
-  //TODO what about brainlife admin ?
-  // if (user.scopes.brainlife) return true;
+  //TODO what about brainlife admin should I keep this ?
```

Comment:
@anibalsolon

Reply:
let's think it through:
what's the information that `user.scopes.brainlife` gives you about the user?

Reply:
the user is the admin of brainlife and its microservices

Reply:
Nick has `user.scopes.brainlife`, is he an admin?
![image](https://github.com/brainlife/auth/assets/562525/3f6dbddd-cec1-4fad-a532-d3e1c94e7543)

Reply:
```json
{
	"scopes": {
		"brainlife": [
			"user"
		]
	}
}
```

Reply:
No he is just a user with basic user privileges. (can create projects, administer his projects etc )

Reply:
Good, please adjust the code accordingly.

Reply:
what's the rationale for the introduction of this check? considering where it is being used.

Reply:
and the rationale of your fix, if it was to check for admin role on brainlife...